sat/cli/bootsys/mgmt_power.py

```diff
@@ -260,6 +260,86 @@ def finish_shutdown(hosts, username, password, ncn_shutdown_timeout, ipmi_timeou
             sys.exit(0)
 
 
+def get_disk_entries(ssh_client, ncn):
+    """
+    Retrieve the current boot order from a node via SSH and identify disk entries.
+
+    Args:
+        ssh_client (paramiko.SSHClient): The SSH client connected to the node.
+        ncn (str): The node associated with the ssh_client.
+
+    Returns:
+        str: The boot number of the first disk entry found.
+
+    Raises:
+        SystemExit: If the command fails, an error is logged and the function exits.
+    """
+    command = "efibootmgr"
+    _, stdout, stderr = ssh_client.exec_command(command)
+    error = stderr.read().decode()
+    if error:
+        LOGGER.error(f'Failed to get current boot order for {ncn}: {error}')
+        raise SystemExit(1)
```

Comment:
The behavior here is not consistent with the rest of the error-handling behavior in this change. This will exit the `sat bootsys` command immediately when we fail to get the boot device list for a single NCN. Elsewhere in the code, you are logging an error and then continuing with the other NCNs and the rest of the command.

My preference would be to change the error log messages to warnings, since we are going to continue even if there is an issue with obtaining and setting next boot using `efibootmgr`, and then continuing rather than raising an exception.

Reply:
Addressed